keras/optimizers.py

```diff
@@ -146,11 +146,16 @@ def get_updates(self, params, constraints, loss):
         moments = [K.zeros(shape) for shape in shapes]
         self.weights = [self.iterations] + moments
         for p, g, m in zip(params, grads, moments):
-            v = self.momentum * m - lr * g  # velocity
+            # Apply learning rate multipliers if needed
+            if p in multipliers:
+                lrm = K.variable(multipliers[p])
```

Comment:
As lr is a K.variable I thought it was safer to multiply lrm*lr both being K.variables... not sure if keras was really happy with number*K.variable